Unity Pitz/Assets/Scripts/Compass.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+using UnityEngine;
+public class Compass : MonoBehaviour
+{
+    public Quaternion MissionDirection;
+    public Transform Arrow;
+    public Transform Missionplace;
```

Comment:
Do these need to be public?

Reply:
No. They did at one point, I just forgot to change them back.